### ISSUE TITLE
NO_CTCP XFLAG should NOT block ACTION CTCP (aka /me)

### ISSUE THREAD

**Issue by KoraggKnightWolf** [DALnet/bahamut#172]
As was brought up in #help and #bahamut it would be beneficial for the NO_CTCP XFLAG to NOT block ACTIONS (aka /me) as those are a universal and fundamental feature of IRC: Another fact that would speak in favor of allowing them is that clients do NOT give out a CTCP response to an ACTION as it is NOT a CTCP request but its own kind of a CTCP ("no response" does NOT include say mIRC scripted on action: events etc, but actual CTCP-based queries).

Regards,

Koragg

**Comment by rscs** [comment]
I agree with this.  I think the original intent was to prevent channel floods of random CTCP stuff, but ACTION is part of everyday conversation.
